src/sss_client/sss_cli.h

```diff
@@ -35,12 +35,15 @@
 
 #include "shared/safealign.h"
 
+#include "config.h"
```

Comment:
Let's have "config.h" as a first include.

(I see there are some places in the code base where this rule isn't followed, to be fixed later)